assets/scripts/admin/users.js

```diff
@@ -38,10 +38,6 @@ function bindButtons(){
     $('.action_usercontrol_button').on('click', function(){
         runAction(this);
     });
-    $('input[name="alltemplates"]').on('switchChange.bootstrapSwitch', function(event, state) {
```

Comment:
Need testing instructions, I think.

Reply:
deleted this as didn't find any files using this js file.
will revert the change.